src/components/SearchResults/ElectronicResourcesLink.tsx

```diff
@@ -15,9 +16,13 @@ interface ElectronicResourcesLinkProps {
 const ElectronicResourcesLink = ({
   bibUrl,
   electronicResources,
+  inSearchResults = true,
```

Comment:
Not my favorite way of doing this but it follows an exist pattern and allows this to be reusable in the search results page and bib details page.

Reply:
what don't you like about it?

Reply:
I'd like the styling for `ElectronicResourcesLink` to be the same on both pages for consistency. That's mostly why.